windowslib/src/main/scala/scala/scalanative/windows/package.scala

```diff
@@ -0,0 +1,33 @@
+package scala.scalanative
+
+import scala.scalanative.unsafe._
+import scala.scalanative.unsigned._
+
+package object windows {
+  type Word      = UShort //uint_16
```

Comment:
Do these definitions really make sense as public members of the `windows` package? Do they correspond to types that are defined in Windows APIs and used everywhere?

Reply:
In my opinion, they do make sense. Windows API uses only its self-defined types everywhere, they're listed on [this page](https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types#pcwstr) 
Sticking to their naming convention allows us to make our API more consistent and eliminates possible type-based bugs. 
However, if placing them in `windows` package is the main concern, we might move them to some object, eg. `windows.WinDef` (since they're defined in  file`WinDef.h`)

Reply:
> In my opinion, they do make sense. Windows API uses only its self-defined types everywhere, they're listed on [this page](https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types#pcwstr)
> Sticking to their naming convention allows us to make our API more consistent and eliminates possible type-based bugs.

OK, sounds good.